src/lib/grid/container.html

```diff
@@ -0,0 +1,3 @@
+<div class="apto-container{{ fluid ? '--fluid' : null }}">
```

Comment:
Since we agreed on SUIT, a proper way to do this would be to always have a class of `apto-container` & when the `fluid` modifier is present, to add it as a separate class in addition to `apto-container`

Reply:
fixed